bin/desi_init_dts.sh

```diff
@@ -3,6 +3,10 @@
 # Sean McManus 2015/11/17
 # Adapted for DESI: Benjamin Weaver 2018-0627
 #
+# DESI Environment
+source /global/common/software/desi/desi_environment.sh test-release
+module use -a ${HOME}/software/${NERSC_HOST}/modulefiles
+module load desispec/my-master
```

Comment:
Do we need "test-release", "${HOME}/software/${NERSC_HOST}/modulefiles", and "my-master", or can that be moved to a more standard release / tag?

Reply:
Is "master" an acceptable release?

Reply:
Actually, how about if I leave the release blank, so it will pick up whatever the default release on datatran is?

Reply:
master is fine, or using datatran default.